tutorials/tutorial1.ipynb

```diff
@@ -164,849 +174,1929 @@
-   "metadata": {},
-   "source": [
-    "## Step 6: Read the validation data and labels and save in .h5 format\n",
-    "\n",
-    "Next we read and save the validation data for the model.\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 10,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "INFO:2020-06-09 20:10:56,268:AtacWorks-bw2h5] Reading intervals\n",
-      "INFO:2020-06-09 20:10:56,410:AtacWorks-bw2h5] Read 1260 intervals\n",
-      "INFO:2020-06-09 20:10:56,410:AtacWorks-bw2h5] Writing data in 2 batches.\n",
-      "INFO:2020-06-09 20:10:56,410:AtacWorks-bw2h5] Extracting data for each batch and writing to h5 file\n",
-      "INFO:2020-06-09 20:10:56,411:AtacWorks-bw2h5] batch 0 of 2\n",
-      "INFO:2020-06-09 20:11:39,943:AtacWorks-bw2h5] Done! Saved to ./Mono.50.2400.val.h5\n"
-     ]
-    }
-   ],
-   "source": [
-    "!python $atacworks/scripts/bw2h5.py \\\n",
-    "           --noisybw dsc.1.Mono.50.cutsites.smoothed.200.bw \\\n",
-    "           --cleanbw dsc.Mono.2400.cutsites.smoothed.200.bw \\\n",
-    "           --cleanpeakbw dsc.Mono.2400.cutsites.smoothed.200.3.narrowPeak.bw \\\n",
-    "           --intervals val_intervals.bed \\\n",
-    "           --out_dir ./ \\\n",
-    "           --prefix Mono.50.2400.val \\\n",
-    "           --pad 5000"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "This produces a .h5 file in the current directory (`Mono.50.2400.val.h5`) containing the validation data for the model.\n",
-    "\n",
-    "\n",
-    "## Step 7: Train and validate a model using the parameters in the given config files\n",
+    "This model has learned a mapping from the 50-cell signal to the 2400-cell signal and peak calls. Given a new 50-cell ATAC-seq track, it can denoise the track and produce high-quality peak calls.\n",
     "\n",
-    "We next train an AtacWorks model to learn a mapping from the noisy (50-cell) ATAC-seq signal to the clean (2400-cell) ATAC-seq signal and peak calls. The two .yaml files that we downloaded into the `configs` directory contain all the parameters that describe how to train the model. `configs/model_structure.yaml` contains parameters that control the architecture of the model and  `configs/config_params.yaml` contains parameters that control the process of training, such as the learning rate and batch size.\n",
+    "See [Tutorial 2](tutorial2.ipynb) for step-by-step instructions on how to apply this trained model to another dataset.\n",
     "\n",
-    "To train the model, we supply the training and validation datasets as well as the two config files."
+    "To change any of the parameters for the deep learning model, you can edit the appropriate parameters in `$atacworks/configs/train_config.yaml` or `$atacworks/configs/model_structure.yaml` and run the command below. Type `atacworks train --help` for an explanation of the parameters.\n"
```

Comment:
The users should never change the default config files in the repository. This could affect all their future experiments. Perhaps it would be better if they copied the config files into their experiment directory in the beginning, then can supply the paths to the copied files and modify within this directory.

Reply:
agree. Changed it to ask the user to copy paste the config and then edit it.